Specs/DateTime/English/DateTimeModelExtendedTypes.json

```diff
@@ -0,0 +1,270 @@
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "next monday 1-3 pm",
+      "TypeName": "datetimeV2.datetimerange",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "(2017-12-11T13,2017-12-11T15,PT2H)",
+            "type": "datetimerange",
+            "start": "2017-12-11 13:00:00",
+            "end": "2017-12-11 15:00:00"
+          }
+        ]
+      }
+    },
+    {
+      "Text": "5-6 pm",
+      "TypeName": "datetimeV2.datetimealt",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "(2017-12-11T17,2017-12-11T18,PT1H)",
+            "type": "datetimealt",
+            "start": "2017-12-11 17:00:00",
+            "end": "2017-12-11 18:00:00",
+            "context": "next monday"
+          }
+        ]
+      }
+    }
+  ]
+},
+{
+  "Input": "Monday 8-9am or 9-10 am works.",
+  "Context": {
+    "ReferenceDateTime": "2017-12-04T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "monday 8-9am",
+      "TypeName": "datetimeV2.datetimerange",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "(XXXX-WXX-1T08,XXXX-WXX-1T09,PT1H)",
+            "type": "datetimerange",
+            "start": "2017-11-27 08:00:00",
+            "end": "2017-11-27 09:00:00"
+          },
+          {
+            "timex": "(XXXX-WXX-1T08,XXXX-WXX-1T09,PT1H)",
+            "type": "datetimerange",
+            "start": "2017-12-04 08:00:00",
+            "end": "2017-12-04 09:00:00"
+          }
+        ]
+      }
+    },
+    {
+      "Text": "9-10 am",
+      "TypeName": "datetimeV2.datetimealt",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "(XXXX-WXX-1T09,XXXX-WXX-1T10,PT1H)",
+            "type": "datetimealt",
+            "start": "2017-12-04 09:00:00",
+            "end": "2017-12-04 10:00:00",
+            "context": "monday"
+          }
+        ]
+      }
+    }
+  ]
+},
+{
+  "Input": "Cortana could try to arrange a Skype call next week on Tuesday or Thursday please?",
+  "Context": {
+    "ReferenceDateTime": "2017-12-04T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "next week on tuesday",
+      "TypeName": "datetimeV2.date",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "2017-12-12",
+            "type": "date",
+            "value": "2017-12-12"
+          }
+        ]
+      }
+    },
+    {
+      "Text": "thursday",
+      "TypeName": "datetimeV2.datetimealt",
+      "Resolution": {
+        "values": [
+          {
+            "timex": "2017-12-14",
+            "type": "datetimealt",
+            "value": "2017-12-14",
+            "context": "next week"
+          }
+        ]
+      }
+    }
+  ]
+},
+{
+  "Input": "Cortana could try to arrange a Skype call next week on Tuesday 9 am or Thursday 1 pm please?",
```

Comment:
"next week on Tuesday 9 am or Thursday 1 pm" should be tagged as ParentText and be a top-level attr of each result.